cli/popper/gha.py

```diff
@@ -313,8 +313,11 @@ def run(self, reuse=False):
 
                 if action_dir:
                     repo_id += '/'
-
-            tag = repo_id + action_dir + ':' + self.env['GITHUB_SHA']
+            
+            if self.env['GITHUB_SHA']:
```

Comment:
the conditional can be removed by doing:

```python
tag = repo_id + action_dir + ':' + self.env.get('GITHUB_SHA', 'na')
```

where `na` is the value assigned with `GITHUB_SHA` is not defined. We could also use `unknown`

Reply:
Oh yes! Thanks.